api/config.py

```diff
@@ -0,0 +1,15 @@
+"""Flask configuration."""
+from os import environ, path
+
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
+load_dotenv(path.join(basedir, ".env"))
+
+
+class Config:
+    """Base config."""
+
+    SECRET_KEY = environ.get("SECRET_KEY")
+    SQLALCHEMY_DATABASE_URI = "sqlite:///bigger_yoshi.db"
```

Comment:
missing environ.get?
[DATABASE_URL](https://devcenter.heroku.com/articles/heroku-postgresql#designating-a-primary-database) is what is set by heroku, it would be natural to use that
```suggestion
    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL", "sqlite:///bigger_yoshi.db")
```

Reply:
Yes  you are correct, I have addressed it now:)